### PR TITLE
update documentation to reflect that -ConfigurationName can be used with SSH

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
Update documentation to reflect code change https://github.com/PowerShell/PowerShell/pull/6603

Version(s) of document impacted
------------------------------
- [X] Impacts 6.1 document
- [ ] Impacts 6.0 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [X] The documented feature was introduced in version (6.1) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Core/Enter-PSSession.md](https://github.com/SteveL-MSFT/PowerShell-Docs/blob/psremoting-configurationname/reference/6/Microsoft.PowerShell.Core/Enter-PSSession.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Core/Invoke-Command.md](https://github.com/SteveL-MSFT/PowerShell-Docs/blob/psremoting-configurationname/reference/6/Microsoft.PowerShell.Core/Invoke-Command.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Core/New-PSSession.md](https://github.com/SteveL-MSFT/PowerShell-Docs/blob/psremoting-configurationname/reference/6/Microsoft.PowerShell.Core/New-PSSession.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C4%5C9%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201804091144315851-2318%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Z2miVG9BtHZziNXTPwnI1Xsu3DxWwhesKV39G5ucsHg%3D&st=2018-04-09T11%3A50%3A55Z&se=2018-05-10T11%3A55%3A55Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by SteveL-MSFT** [comment]
@sdwheeler it's fine if you don't revert as my new PR should overwrite it anyways

**Comment by doctordns** [comment]
Hi

Thanks for the mail.

Can I order 6 of the ‘La 50/50’ and 6 of the ‘Clos de l’Abbaye’ rouge.

Thanks.

File: reference/6/Microsoft.PowerShell.Core/Enter-PSSession.md [hunk 1]
```diff
@@ -304,6 +305,9 @@ Specifies the session configuration that is used for the interactive session.
 Enter a configuration name or the fully qualified resource URI for a session configuration.
 If you specify only the configuration name, the following schema URI is prepended: http://schemas.microsoft.com/powershell.
 
+When used with SSH, this specifies the subsystem to use on the target as defined in sshd_config.
```
Comment (PaulHigin) [inline reference/6/Microsoft.PowerShell.Core/Enter-PSSession.md]: I feel that ConfigurationName parameter should not be used to specify an SSH session subsystem.  We should reserve this for when we add session configuration support to SSH.

**Pull request merged**